lib/PuppeteerSharp/CDPSession.cs

```diff
@@ -101,26 +101,18 @@ internal CDPSession(IConnection connection, TargetType targetType, string sessio
         /// <returns>The task.</returns>
         public async Task<T> SendAsync<T>(string method, dynamic args = null)
         {
-            var content = await SendAsync(method, true, args).ConfigureAwait(false);
-            return JsonConvert.DeserializeObject<T>(content);
-        }
+            JObject content = await SendAsync(method, args);
 
-        /// <summary>
-        /// Protocol methods can be called with this method.
-        /// </summary>
-        /// <param name="method">The method name</param>
-        /// <param name="args">The method args</param>
-        /// <returns>The task.</returns>
-        public Task<dynamic> SendAsync(string method, dynamic args = null) => SendAsync(method, false, args ?? new { });
+            return content.ToObject<T>();
+        }
 
         /// <summary>
         /// Protocol methods can be called with this method.
         /// </summary>
         /// <param name="method">The method name</param>
         /// <param name="args">The method args</param>
-        /// <param name="rawContent"></param>
         /// <returns>The task.</returns>
-        public async Task<dynamic> SendAsync(string method, bool rawContent, dynamic args = null)
+        public async Task<JObject> SendAsync(string method, dynamic args = null)
```

Comment:
JObject represents json object `{ ... }`, JArray represents `[ ... ]`, I think you should use `JToken` since it represents any json

Reply:
I did consider JToken for the SenAsync but I'm not sure you can ever get a response for a type like JArray from the devtools api ?

Reply:
ok, I can buy that